src/clr-angular/button/button-group/button-group.ts

```diff
@@ -25,9 +38,22 @@ export class ClrButtonGroup {
   constructor(
     public buttonGroupNewService: ButtonInGroupService,
     private elementRef: ElementRef,
+    private toggleService: ClrPopoverToggleService,
+    @Inject(UNIQUE_ID) public popoverId: string,
     public commonStrings: ClrCommonStringsService
   ) {}
 
+  public popoverPosition: ClrPopoverPosition = {
```

Comment:
what's the difference between this and...

```
public popoverPosition = ClrPopoverPositions['bottom-left'];
```

Reply:
Yup, this was the original before I abstracted the positions, will fix up.